betaflight

```diff
@@ -0,0 +1,215 @@
+
+
+
+# Betaflight / STM32F405 (S405) 4.0.0 Apr  3 2019 / 14:30:44 (22b9f3453) MSP API: 1.41
+
+board_name HGLRCF4V6
+manufacturer_id HGL
```

Comment:
This is not a valid manufacturer id. Please read and follow https://github.com/betaflight/betaflight/blob/master/unified_targets/docs/CreatingAUnifiedTarget.md if you want to have unified targets added to Betaflight.